cms/__init__.py

```diff
@@ -60,21 +60,28 @@
 LANG_CPP = "cpp"
 LANG_CS = "cs"
 LANG_PASCAL = "pas"
-LANG_PYTHON = "py"
+LANG_PYTHON2 = "py2"
+LANG_PYTHON3 = "py3"
 LANG_PHP = "php"
 LANG_JAVA = "java"
 
+# Python 3 py_compile outputs to __pycache__/filename.<PYTHON3_COMPILE_NAME>.pyc
+# Using a '*' in this place does not appear to work correctly.
+# Format seems standard as cpython-<MAJOR><MINOR> for cpython.
+PYTHON3_COMPILE_NAME = "cpython-%d%d" % (3, 4)
```

Comment:
Good point, used your original idea, didn't bother fetching the 3 though because that will never change (especially as we call 'python3' directly)